lib/inspec/cli.rb

```diff
@@ -99,36 +99,32 @@ def check(path) # rubocop:disable Metrics/AbcSize
       puts JSON.generate(result)
     else
       %w{location profile controls timestamp valid}.each do |item|
-        puts format('%-12s %s', item.to_s.capitalize + ':',
-                    mark_text(result[:summary][item.to_sym]))
+        prepared_string = format('%-12s %s',
+                                 (item.to_s.capitalize + ':'),
```

Comment:
This isn't your change, but we should avoid string concatenation whenever possible. That `+ ':'` could just be part of the format string for free.